presto-plugin-toolkit/src/test/java/com/facebook/presto/plugin/base/security/TestFileBasedAccessControl.java

```diff
@@ -25,6 +25,7 @@
 import java.io.IOException;
 import java.util.Optional;
 
+import static com.facebook.presto.plugin.base.security.TestAllowAllAccessControl.checkEverythingImplemented;
```

Comment:
This means, the method should be defined in some shared util class (might be package private)